### PR TITLE
Admin status page

### PR DESCRIPTION
**Pull request by anusha-dey**
`docker-compose exec adminstatus python3 /render_adminpage.py openldap test.txt` should create the file test.txt with the html table of users and expiry dates. `docker-compose exec adminstatus cat test.txt` to view the file

### CODE REVIEW

File: adminstatus/render_adminpage.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            time_delta = (user_expire - datetime.datetime.now()).days
+            # push style to class
+            if time_delta < 0:
+                user_status="danger"
+            elif 0 <= time_delta < 7:
+                user_status="warning"
+            else:
+                user_status="success"
+        except:
+            print("Improper configuration for user: {}".format(user_name))
```
Comment (IntegralProgrammer) [inline adminstatus/render_adminpage.py:65]: If we have an improperly configured user, can we highlight this entry in **red** and for the expiration date, display **Improperly Configured**? Also, if there is at least one improperly configured user, a warning of this should be displayed at the top of the rendered page and the **Confirm Review** button should be disabled.

File: adminstatus/templates/adminpage.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Bootstrap Example</title>
```
Comment (IntegralProgrammer) [inline adminstatus/templates/adminpage.html]: Change `title` to **Authorized Users**

File: adminstatus/render_adminpage.py [hunk 3]
```diff
@@ -57,26 +65,32 @@ def render_adminpage(output_filename):
...
-        user = dict(name=user_name, expire=user_expire, status=user_status)
-        users.append(user)
-        users_to_send.append([user_name, expire_string])
+        finally:
+            user = dict(name=user_name, expire=user_expire, status=user_status)
+            users.append(user)
+            users_to_send.append([user_name, expire_string])
 
     # convert to JSON to be sent
     user_info = json.dumps(users_to_send)
```
Comment (IntegralProgrammer) [inline adminstatus/render_adminpage.py]: Unfortunately, calling `json.dumps` emits characters that do not play nicely when written into a HTML document and the complete data is not sent. As a fix, I would suggest Base64 encoding the output from `json.dumps` and placing it in between double-quotes.

Comment (anusha-dey) [reply adminstatus/render_adminpage.py]: The data should be sent now, I converted the output from json.dumps to a bytes-like object and then encoded with base64 like you suggested

**Pull request merged**